### PR TITLE
Updated instructions with the public Grafeas server image

### PR DESCRIPTION
**Pull request by aysylu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,16 @@
...
-2. Build the images and run the containers:
+1. Pull the Grafeas Server image:
 
    ```bash
-   docker-compose build
-   docker-compose up
+   docker pull us.gcr.io/grafeas/grafeas-server:0.1.0
+   ```
+
+1. Build the images and run the containers:
```
Comment (clrprod) [inline README.md:16]: Should this be 3?

File: README.md [hunk 2]
```diff
@@ -7,11 +7,16 @@
 To start the standalone instance of Grafeas with PostgreSQL, follow these steps:
 
 1. Install [Docker Compose](https://docs.docker.com/compose/install/).
-2. Build the images and run the containers:
```
Comment (clrprod) [inline README.md:10]: Should this be 2?

File: server/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/grafeas/grafeas-server
+FROM us.gcr.io/grafeas/grafeas-server:0.1.0
```
Comment (clrprod) [inline server/Dockerfile:1]: Is there a travis test that makes sure docker build works here?

**Review by clrprod — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (aysylu) [reply README.md:10]: markdown renders this correctly, just makes it easier if these are ever re-ordered or a new step is added to not have to re-number the whole list

File: README.md (hunk 1 above)
Comment (aysylu) [reply README.md:16]: ditto here for markdown doing the correct numbering

File: server/Dockerfile (hunk 3 above)
Comment (aysylu) [reply server/Dockerfile:1]: Added a CircleCI test, for consistency with Grafeas setup, that does `docker-compose up` and curls an endpoint.

**Comment by aysylu** [comment]
The integration tests now pass: https://circleci.com/gh/aysylu/grafeas-pgsql/14

**Pull request merged**